Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -528,7 +530,7 @@ void MainWindow::load_plugin(QString fileName, bool blacklisted)
       {
         if ( plugin_blacklist.contains(name) ){
           qDebug("### Ignoring plugin \"%s\".", qPrintable(fileName));
-          return;
+          return true;
```

Comment:
If the plugin is blacklisted, it is not loaded, and I think the function should return `false`, otherwise the plugin will be listed in the loaded plugins.